packages/dd-trace/test/log.spec.js

```diff
@@ -265,7 +265,10 @@ describe('log', () => {
     it('should log a deprecation warning', () => {
       log.deprecate('test', 'message')
 
-      expect(console.error).to.have.been.calledWith('message')
+      expect(console.error).to.have.been.calledOnce
+      const consoleErrorArg = console.error.getCall(0).args[0]
+      expect(typeof consoleErrorArg).to.be.eq('object')
+      expect(consoleErrorArg.message).to.be.eq('message')
```

Comment:
This changes the behaviour of `log.deprecate`, doesn't it?
`console.error` is now called by `{ 'message': 'message' }` instead of with `'message'`

Reply:
Yes, now `log.deprecate` emits the message via `log:error` channel so it is handled like the rest: calling `console.error` with `new Error('message')`
BTW, is somebody using `log.deprecate` method? I've be tempted to remove it.

Reply:
It was `error` before too, so not sure why the behaviour changes 🤔 It looks like it should always have been an object.

> BTW, is somebody using log.deprecate method? I've be tempted to remove it.

It's not used right now as we've removed all deprecated methods. Could be useful in the future, although I would argue that logging is not the best place to do this.